src/main/java/pubsub/eventmodel/CarModel.java

```diff
@@ -0,0 +1,65 @@
+package pubsub.eventmodel;
+
+import lombok.Data;
+
+@Data
+public class CarModel {
```

Comment:
no need to create inner class

Reply:
After making the changes in the model I was facing an issue so I reverted the changes and I am currently working on the same